cpp/src/rolling/range_window_bounds.cpp

```diff
@@ -58,8 +61,14 @@ struct range_scalar_constructor {
     return std::make_unique<fixed_point_scalar<T>>(
       static_cast<fixed_point_scalar<T> const&>(range_scalar_));
   }
-};
 
+  template <typename T, CUDF_ENABLE_IF(std::is_floating_point_v<T>())>
+  std::unique_ptr<scalar> operator()(scalar const& range_scalar_) const
+  {
+    return std::make_unique<fixed_point_scalar<T>>(
```

Comment:
`fixed_point_scalar` for floating-point?
```suggestion
    return std::make_unique<fixed_point_scalar<T>>(
```

Reply:
I'm re-checking. I thought I'd fixed that. Thanks for finding that.

Reply:
Yep. This is messed up. I'll post a fix. 
Odd that this passed tests. I would have thought this would result in ambiguity.

Reply:
Fixed now.